src/Configuration/MessageBrokerConfigurationKeys.cs

```diff
@@ -27,5 +27,12 @@ public class MessageBrokerConfigurationKeys
         /// </summary>
         [ConfigurationKeyName("exportRequestPrefix")]
         public string ExportRequestPrefix { get; set; } = "md.export.request";
+
+        /// <summary>
+        /// Gets or sets the topic for publishing workflow requests.
+        /// Defaults to `md_workflow_request`.
+        /// </summary>
+        [ConfigurationKeyName("exportRequestPrefix")]
```

Comment:
ConfigurationKeyName needs changing too

Reply:
Looks like victor is using these two within his PR 
![image](https://user-images.githubusercontent.com/56344499/167799640-8efe5690-06ea-4c8e-9c50-b2905c92f327.png)